### PR TITLE
Heroku Update Command.

### PR DESCRIPTION
**Pull request by pallavbs2**

### CODE REVIEW

File: appuselfbot.py [hunk 1]
```diff
@@ -408,36 +408,30 @@ def check(msg):
...
-                except:
-                    pass
-            await ctx.send(bot.bot_prefix + 'There is an update available. Downloading update and restarting (check your console to see the progress)...')
+
+@bot.command()
+async def update(self, ctx):
+    '''Auto Update command, checks if you have latest version
+    Use tags github-token to find out how to set up this token'''
+    git = self.bot.get_cog('Git')
+    if not await git.starred('PallavBS/Discord-Selfbot-1'): return await ctx.send('**This command is disabled as the user have not starred <https://github.com/PallavBS/Discord-Selfbot-1>**')
```
Comment (chenzw95) [inline appuselfbot.py]: Holding the update command hostage by requiring that users star your github repo is a very quick way to ensure that this PR does not get through.

File: settings/config.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (chenzw95) [inline settings/config.json]: Please do not contaminate PRs with configuration files from your own environment.

File: settings/optional_config.json [hunk 3]
```diff
@@ -4,11 +4,11 @@
     "show_search_term": "True",
     "customcmd_color": "27007A",
     "rich_embed": "on",
-    "default_status": "idle",
+    "default_status": "dnd",
```
Comment (chenzw95) [inline settings/optional_config.json]: Please do not contaminate PRs with configuration files from your own environment.

**Comment by astronautlevel2** [comment]
Hey there,

Unfortunately I'm going to have to close this PR for a number of reasons:

1. You changed/added far more than just updating on heroku, and due to the huge volume of commits without helpful commit messages, it's hard to tell exactly what, and I don't think anyone on the team feels comfortable merging it.

2. The actual heroku updating code has some concerning parts, such as forcing the user to star your repo, or pulling from your fork instead of from upstream.

3. Your PR has a bunch of configuration files from your environment.

4. It failed CI, and I don't feel comfortable merging broken code.

If you want to rewrite it and open a PR for just this command, and fix the issues I described with it, we'd be happy to merge.

**Pull request closed without merging**